asset_dashboard/views.py

```diff
@@ -39,9 +41,37 @@ def get_context_data(self, **kwargs):
         # send the form to a modal in this view
         form = ProjectForm()
         context['form'] = form
+
+        # need all of the Sections and Categories for filtering the table
+        context['sections'] = [s.name for s in Section.objects.all().order_by('name')]
+        context['categories'] = [c.slug for c in ProjectCategory.objects.all().order_by('slug')]
+
         return context
 
 
+class ProjectListJson(BaseDatatableView):
+    model = Project
+    columns = ['name', 'description', 'section_owner', 'category', 'id']
+    order_columns = ['name', 'description', 'section_owner__name', 'category__category']
+    max_display_length = 500
+
+    def filter_queryset(self, qs):
+        search = self.request.GET.get('search[value]', None)
+        section = self.request.GET.get('columns[2][search][value]', None)
```

Comment:
hmm... is there another way to name these so we don't have to do "columns[2]"?

Reply:
I'm pretty sure that's the way [the DataTables jquery plugin](https://datatables.net/examples/index) formats the query string for the request. 

If a user filters or searches the table, the plugin sends an ajax request to the server. The request includes query strings that are formatted like`columns[2]`.

Here's an example of what this query string looks like, from a test: https://github.com/datamade/ccfp-asset-dashboard/blob/c61812229a781f3b1c971fe8c7aa27801be6c54c/tests/test_views.py#L33

I'll look into the library and see if there's a way to make it more explicit.